os/include/tinyara/fs/ioctl.h

```diff
@@ -389,6 +389,16 @@
 #define _SPIIOCVALID(c)   (_IOC_TYPE(c) == _SPIBASE)
 #define _SPIIOC(nr)       _IOC(_SPIBASE, nr)
 
+
+#define _LCDIOCBASE     (0x1100) /* LCD character driver ioctl commands */
```

Comment:
_SLCDIOCBASE already defined as 0x1100 already.
Hence you should put new base value to line 104